src/app/public/modules/repeater/repeater-item.component.ts

```diff
@@ -209,6 +254,97 @@ export class SkyRepeaterItemComponent implements OnDestroy, OnInit {
     this.updateForExpanded(direction === 'up', true);
   }
 
+  // Cycle backwards through interactive child elements.
+  // If user reaches the beginning, focus on parent item.
+  public onArrowLeft(event: KeyboardEvent): void {
+    const focusableChildren = this.adapterService.getFocusableChildren(
+      this.itemRef.nativeElement,
+      {
+        ignoreTabIndex: true
+      }
+    );
+    if (focusableChildren.length > 0) {
+      if (this.childFocusIndex > 0) {
+        this.childFocusIndex--;
+      } else if (this.childFocusIndex === 0) {
+        this.childFocusIndex = undefined;
+      }
+    }
+    event.stopPropagation();
+    event.preventDefault();
+  }
+
+  // Cyle forward through interactive child elements.
+  // If user reaches the end, do nothing.
+  public onArrowRight(event: KeyboardEvent): void {
+    const focusableChildren = this.adapterService.getFocusableChildren(
+      this.itemRef.nativeElement,
+      {
+        ignoreTabIndex: true
+      }
+    );
+    if (focusableChildren.length > 0) {
+      if (this.childFocusIndex < focusableChildren.length - 1) {
+        this.childFocusIndex++;
+      } else if (this.childFocusIndex === undefined) {
+        this.childFocusIndex = 0;
+      }
+    }
+    event.stopPropagation();
+    event.preventDefault();
+  }
+
+  public onArrowUp(event: KeyboardEvent): void {
+    this.childFocusIndex = undefined;
+    this.repeaterService.focusPreviousListItem(this);
+    event.preventDefault();
+    event.stopPropagation();
+  }
+
+  public onArrowDown(event: KeyboardEvent): void {
+    this.childFocusIndex = undefined;
+    this.repeaterService.focusNextListItem(this);
+    event.preventDefault();
+    event.stopPropagation();
+  }
+
+  // The dropdown compoment supports enter, space, and arrow keys. This event listner will
+  // prevent those keyboard controls from bubbling up to tree view component.
```

Comment:
I would avoid mentioning external libraries if we can avoid it. This method should ideally be beneficial in its own right.

Reply:
Removed.